### PR TITLE
feat: add Swiss National Supercomputing Center

### PR DESCRIPTION
**Pull request by hairmare**
Is adding the CSCS like so ok or would it make more sense to put it in universities?

There are a bunch of other `eth-*` orgas (ie. @SwissDataScienceCenter & @ethz-asl) that I'd like to add as well... Maybe a `Universities_Federal` category would make sense?

I'll rebase this PR if you'd rather go in that direction. Let me know how you'd prefer ETH stuff to be added.

### CODE REVIEW

File: data-gathering/github_repos.json [hunk 1]
```diff
@@ -99,7 +99,8 @@
 				{"name": "Bundesamt für Statistik","orgs": ["BFS-SHS-MSAS","ogdch"]},
 				{"name": "Bundeskanzlei","orgs": ["swiss"]},
 				{"name": "MeteoSwiss","orgs": ["MeteoSwiss-APN"]},
-				{"name": "swisstopo","orgs": ["geoadmin"]}
+				{"name": "swisstopo","orgs": ["geoadmin"]},
+				{"name": "Swiss National Supercomputing Center","orgs": ["eth-csc"]}
```
Comment (CodeOneTwo) [inline data-gathering/github_repos.json:103]: @maemst please review

**Comment by CodeOneTwo** [comment]
The institutions are created manually. I think it would be best to group the organizations in the universities under eth. But maybe @maemst has a better idea?

**Comment by hairmare** [comment]
Some cases like @SwissDataScienceCenter are collaborations between ETHZH and EPFL. I guess we'd want to think about that as well.

**Comment by maemst** [comment]
I changed "Universities" to "ResearchAndEducation". I suggest to create a new institution for every independent research organization such as @SwissDataScienceCenter or CSCS and to aggregate direct ETHZ units like @ethz-asl with a new institution "ETH Zürich". If you agree please adapt your code changes and commit them again.

**Comment by hairmare** [comment]
I moved CSCS to `ResearchAndEducation` and rebased onto master.

This will now conflict with #25 and I'll rebase the one we merge last again.

**Pull request merged**